templates/availability/tab-session-length.phtml

```diff
@@ -1,5 +1,11 @@
 <session-length inline-template>
     <div>
+        <div class="alert alert-warning" style="margin-bottom: 0; margin-top: .5rem">
+            <span class="dashicons dashicons-info"></span>
+            <span>
+                The "day" and "week" durations for session lengths are coming soon.

```

Comment:
What about i18n?

Reply:
Conclusion: Leave current strings as is. In the future, use JS to *translate all human-readable strings*.